src/conn/conn_capacity.c

```diff
@@ -34,14 +34,30 @@ __capacity_config(WT_SESSION_IMPL *session, const char *cfg[])
     uint64_t chunkcache, total;
 
     conn = S2C(session);
-    chunkcache = total = 0;
+    cap = &conn->capacity;
 
+    chunkcache = total = 0;
     WT_RET(__wt_config_gets(session, cfg, "io_capacity.total", &cval));
     if (cval.val != 0) {
         if (cval.val < WT_THROTTLE_MIN)
             WT_RET_MSG(session, EINVAL, "total I/O capacity value %" PRId64 " below minimum %d",
               cval.val, WT_THROTTLE_MIN);
-        total = (uint64_t)cval.val;
+        cap->total = total = (uint64_t)cval.val;
+    }
+
+    WT_RET(__wt_config_gets(session, cfg, "io_capacity.fsync_maximum_wait_period", &cval));
+    if (cval.val != 0) {
+        if (cval.val < 0)
+            WT_RET_MSG(session, EINVAL,
+              "io_capacity.fsync_maximum_wait_period value %" PRId64 " cannot be less than 0",
+              cval.val);
```

Comment:
I don't think we need this check nor the message, because the config system has already made the check.  Since we have:
```
        Config('fsync_maximum_wait_period', '0', r'''
            max waiting period for background fsync. If the written threshold is not met in this
            time, a background fsync is done.''',
            min='0', max='60'),
```
in `dist/api_data.py`, with a `min` set to 0.

Reply:
I have removed this useless check, thanks.